src/Object/Contact.php

```diff
@@ -469,49 +460,31 @@ public static function photoMenu(array $contact, $uid = 0)
 	}
 
 	/**
+	 * @brief Returns ungrouped contact count or list for user
+	 *
 	 * Returns either the total number of ungrouped contacts for the given user
 	 * id or a paginated list of ungrouped contacts.
 	 *
-	 * @brief Returns ungrouped contact count or list for user
+	 * @param int $uid   uid
+	 * @param int $start optional, default 0
+	 * @param int $count optional, default 0
 	 *
-	 * @param int $uid
-	 * @param int $start
-	 * @param int $count
 	 * @return array
 	 */
 	public static function getUngroupedList($uid, $start = 0, $count = 0)
 	{
 		if (!$count) {
-			$r = q(
-				"SELECT COUNT(*) AS `total`
-				 FROM `contact`
-				 WHERE `uid` = %d
-				 AND `self` = 0
-				 AND `id` NOT IN (
-					SELECT DISTINCT(`contact-id`)
-					FROM `group_member`
-					WHERE `uid` = %d
-				) ", intval($uid), intval($uid)
-			);
+			$fields = array('COUNT(*) AS `total`');
```

Comment:
I don't think that this does work. ```dba::select``` currently only works with real field names. I guess you have to use ```dba::p``` here instead.